data/language/english_uk.txt

```diff
@@ -4039,8 +4038,20 @@ STR_5732    :Anisotropic
 STR_5733    :Use NN scaling at integer scales
 # tooltip for tab in options window
 STR_5734    :{SMALLFONT}{BLACK}Rendering
-STR_5735    :Network Status
-STR_5736    :Player
```

Comment:
I don't think there's usually this many additions all at once, but I think that the language files in general will probably end up needing to be refactored once the entire game is disassembled.  It's possible that's already the plan, I don't remember for sure.